app/controllers/wms_controller.rb

```diff
@@ -1,4 +1,6 @@
 class WmsController < ApplicationController
+  skip_before_action :verify_authenticity_token
```

Comment:
I'm a little uneasy about this line. Is there a change in Leaflet that requires this or is this a Rails thing?